Wdb 10/setup login page

kibartas opened the pull request.

Commit:
[WDB-10]
create dummy LoginForm component

Commit:
[WDB-10]
add loginPage icons to the project

Commit:
[WDB-10]
remove fake material-ui and add real @material-ui/core

Commit:
[WDB-10]
add link to Roboto font to index.html for material-ui

Commit:
[WDB-10]
add base style to root html and body components

Commit:
[WDB-10]
create LoginPage container

Commit:
[WDB-10]
move product-image.svg to assets/generic

kibartas commented on frontend/public/index.html:
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
```
Adds Roboto font

AArlauskas replied:
https://www.youtube.com/watch?v=uc6f_2nPSX8

kibartas commented on frontend/src/containers/LoginPage.js:
```diff
@@ -1,8 +1,31 @@
+        direction="row"
+        alignItems="center"
+      >
+        <Grid item />
```
Dummy Grid items so that the layout looks prettier (imo)

Commit:
[WDB-10]
delete trash files

Commit:
[WDB-10]
add CssBaseline (replacemenet for style.css)

Commit:
[WDB-10]
remove styling from index.html (CssBaseline takes care of that now)

Commit:
[WDB-10]
add a touch of responsive design to LoginPage

AArlauskas commented on frontend/src/containers/LoginPage.js:
```diff
@@ -1,8 +1,31 @@
+      <Grid
+        container
+        justify="space-around"
+        style={{ height: '100vh' }}
```
Are we going with inline styles and not separate css or scss files?

tmaciulis22 replied:
I think we should agree on what would be the best choice for this project

kibartas replied:
I'd say inline, but we can do separate if you guys wanna

tmaciulis22 commented on frontend/src/containers/LoginPage.js:
```diff
@@ -1,8 +1,31 @@
+        direction="row"
+        alignItems="center"
+      >
+        <Hidden smDown>
```
🧡

tmaciulis22 reviewed (changes requested):
Let's agree on what we should do with custom styles. Inline or css files?

Commit:
Merge branch 'master' into WDB-10/Setup-login-page

Commit:
[WDB-10]
change file structure in containers folder to accomodate styling in another file

Commit:
[WDB-10]
create styling file for LoginPage.js

tmaciulis22 commented on frontend/src/containers/LoginPage/styles.css:
```diff
@@ -0,0 +1,3 @@
+.root {
```
Is using `.root` a good choice here? Maybe it would be better to make this style specific to a className `side-image`?

kibartas replied:
This style is for the root grid element. That's why it's named root. It's not for the side-image
